src/main/java/com/programmers/voucher/dto/VoucherRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.programmers.voucher.dto;
+
+public record VoucherRequestDto(String voucherType, long discountAmount) {
+
+    public VoucherRequestDto {
+    }
+
+    public String getVoucherType() {
+        return voucherType;
+    }
+
+    public long getDiscountAmount() {
+        return discountAmount;
+    }
+
+}
```

Comment:
record라서 getter, 기본 생성자 전부 필요 없는 코드인 것 같아요.

```suggestion
public record VoucherRequestDto(String voucherType, long discountAmount) {
}
```

Reply:
07bfe5da2a22a6995e6463a469291431383b6767
해당 사항 반영하였습니다!